source/pug/templates/master.pug

```diff
@@ -13,17 +13,11 @@ html
     | {% load settings_value %}
   head
     meta(charset=`utf-8`)
-
     meta(name=`viewport`, content=`width=device-width, initial-scale=1`)
-    meta(name=`description`, content=`Mozilla is fueling a movement for a healthy Internet. See how a network of dedicated leaders and activists are working for a better Internet for all, connecting us to each other and to a better future.`)
+    meta(property=`og:type`, content=`website`)
```

Comment:
there should be no changes necessary to the pug template. All tags should come from the modulepage template, so if that lacks these tags, turn that into 
```
{% block socialMetadata %}
    {% meta_tags %}
    <meta name='og:type' content='website'>
    <meta name='og:locale' content='en_US'>
{% endblock %}
```